atomic_reactor/plugin.py

```diff
@@ -227,6 +221,26 @@ def run(self, keep_going=False, buildstep_phase=False):
                 plugin_is_allowed_to_fail = plugin_request['is_allowed_to_fail']
             except (TypeError, KeyError):
                 plugin_is_allowed_to_fail = getattr(plugin_class, "is_allowed_to_fail", True)
+            plugin_conf = plugin_request.get("args", {})
+            plugins_data.append((plugin_name, plugin_class, plugin_conf, plugin_is_allowed_to_fail))
```

Comment:
I'm not a huge fan of having structured data in tuples. Could we use NamedTuple or dict?